src/main/webapp/package.json

```diff
@@ -17,7 +17,8 @@
     "core-js": "^3.6.5",
     "date-fns": "^2.16.1",
     "vue": "^2.6.12",
-    "vuelidate": "^0.7.5"
+    "vuelidate": "^0.7.5",
+    "moment": "^2.27.0"
```

Comment:
date-fns are already used, please use them instead

Reply:
I am doing something wrong here. Somehow format is undefined after switching to date-fns. Could you check this?